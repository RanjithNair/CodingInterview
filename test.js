describe('Coding Tests', () => {
  it('Increase the age attribute by 2', () => {
    let obj = [
      {
        name: 'Roger Federer',
        age: 37
      },
      {
        name: 'Rafael Nadal',
        age: 32
      },
      {
        name: 'Novak Djokovic',
        age: 30
      }
    ]
    const modifiedObj = []
    expect(obj).toMatchSnapshot()
  })

})
