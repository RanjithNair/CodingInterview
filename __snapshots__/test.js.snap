// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Coding Tests Increase the age attribute by 2 1`] = `
Array [
  Object {
    "age": 39,
    "name": "Roger Federer",
  },
  Object {
    "age": 34,
    "name": "Rafael Nadal",
  },
  Object {
    "age": 32,
    "name": "Novak Djokovic",
  },
]
`;
